board/safety.h

```diff
@@ -250,6 +252,7 @@ const safety_hook_config safety_hook_registry[] = {
   {SAFETY_NISSAN, &nissan_hooks},
   {SAFETY_NOOUTPUT, &nooutput_hooks},
   {SAFETY_HYUNDAI_LEGACY, &hyundai_legacy_hooks},
+  {SAFETY_STELLANTIS, &stellantis_hooks},
```

Comment:
we'll probably merge this before getting one to test ourselves, so let's gate this behind the debug flag for now